service/controller/resource/configmap/desired.go

```diff
@@ -51,7 +52,14 @@ func (r *Resource) newConfigMaps(customResource v1alpha1.KVMConfig) ([]*apiv1.Co
 			return nil, microerror.Maskf(notFoundError, fmt.Sprintf("node index for master (%q) is not available", node.ID))
 		}
 
-		template, err := r.cloudConfig.NewMasterTemplate(customResource, certs, node, keys, nodeIdx)
+		ignition := cloudconfig.IgnitionTemplateData{
+			CustomObject: customResource,
+			ClusterCerts: certs,
+			ClusterKeys:  keys,
+			// Images?
```

Comment:
Images!

You can check `aws-operator` on how to actually get the images here. We can also hang tomorrow if you want.

Reply:
I checked there but I think this works differently for KVM?

Reply:
It should be reasonably similar :)

Reply:
It was :)